### PR TITLE
Full lint for google3 re-import.

### PR DESCRIPTION
**Pull request by adarob**

### CODE REVIEW

File: magenta/models/latent_transfer/interpolate_joint.py [hunk 1]
```diff
@@ -23,15 +23,15 @@
...
-import tensorflow as tf
+import os
 
 from magenta.models.latent_transfer import common
 from magenta.models.latent_transfer import common_joint
 from magenta.models.latent_transfer import model_joint
+import numpy as np
+import tensorflow as tf
 
+join = os.path.join
```
Comment (iansimon) [inline magenta/models/latent_transfer/interpolate_joint.py]: There are few enough calls that I would prefer to just use the full `os.path.join` for each call.

File: magenta/models/latent_transfer/sample_wavegan.py [hunk 2]
```diff
@@ -21,14 +21,15 @@
...
-from os.path import join
+import operator
+import os
 
 import numpy as np
 from scipy.io import wavfile
 import tensorflow as tf
 from tqdm import tqdm
 
+join = os.path.join
```
Comment (iansimon) [inline magenta/models/latent_transfer/sample_wavegan.py]: Same as above.

File: magenta/models/latent_transfer/train_joint.py [hunk 3]
```diff
@@ -27,13 +27,12 @@
 import importlib
 from os.path import join
```
Comment (iansimon) [inline magenta/models/latent_transfer/train_joint.py]: Why is from `os.path import join` used here but not in the previous files?  I prefer the full `os.path.join` either way.

File: magenta/protobuf/music_pb2.py [hunk 4]
```diff
@@ -3,11 +3,13 @@
...
 
 import sys
-_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
+
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import message as _message
 from google.protobuf import reflection as _reflection
 from google.protobuf import symbol_database as _symbol_database
+
+_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
```
Comment (iansimon) [inline magenta/protobuf/music_pb2.py]: How sure are you that this doesn't break anything?  Is there any chance the above imports were somehow reading this variable?

File: magenta/models/latent_transfer/train_joint.py (hunk 3 above)
Comment (adarob) [reply magenta/models/latent_transfer/train_joint.py]: Weird. I'm not sure how the linter missed this...

File: magenta/protobuf/music_pb2.py (hunk 4 above)
Comment (adarob) [reply magenta/protobuf/music_pb2.py]: reverted

**Pull request merged**